test/models/user_test.exs

```diff
@@ -15,6 +15,7 @@ defmodule Yggdrasil.UserTest do
   @missing_username           Map.drop(@valid_attrs, [:username])
   @missing_password           Map.drop(@valid_attrs, [:password])
   @missing_password_conf      Map.drop(@valid_attrs, [:password_confirmation])
+  @username_spaces            Map.update!(@valid_attrs, :username, fn username -> "username with spaces" end)
```

Comment:
@ChristianFlinchum look at using the map update syntax. [Here](http://elixir-lang.org/docs/stable/elixir/Kernel.SpecialForms.html#%25%7B%7D/1) is a link to it. Scroll down a little to the header that says `Update syntax`